### ISSUE TITLE
KF5 adaptation

### ISSUE THREAD

**Issue by arcan1s** [zaufi/kate-cpp-helper-plugin#23]

**Comment by zaufi** [comment]
Yeah, sure I have plans, but unfortunately it happens not early than Python bindings for KF5 will be available and Pate plugin will be ported. Cuz except this plugin in my daily job I use a lot of Python plugins for `kate` and can't live w/o them... otherwise `kate` build w/ KF5 is useless for me.
